tools/bin/commands/util/openshift_funcs

```diff
@@ -1,11 +1,66 @@
 #!/bin/bash
 
+OC_MIN_VERSION=3.9.0
+
+compare_version_part() {
+    local test=$1
+    local min=$2
+
+    test=$(($test + 0))
+    min=$(($min + 0))
```

Comment:
If we could avoid bashism, that would be awesome. Actually we try to avoid shell specific constructs as much ass possible.